packages/atomic/src/components/common/search-box/search-text-area.tsx

```diff
@@ -81,7 +81,7 @@ export const SearchTextArea: FunctionalComponent<Props> = ({
   ...defaultInputProps
 }) => (
   <div class="grow flex overflow-hidden">
-    <div class="grow-wrap grow grid overflow-hidden">
+    <div part="textarea-wrapper" class="grow grid overflow-hidden">
```

Comment:
I would prefer to avoid names like "wrapper" when possible, and make parts as intuitive to implementers as possible.

What is the exact purpose of this element? Is it to expand the textarea? If so, maybe this could be `textarea-expander` or something.

Reply:
not sure what you mean by `-self` properties.

The purpose of this element is for the auto-expanding trick with the ::after pseudo-element to work. The value of the text-area is set in an attribute on the parent, then with CSS we're setting the `content` of the ::after element to match that value.

I don't mind renaming the part to textarea-expander at all though.